source/ui/beamline/AMControlConnectedLEDView.cpp

```diff
@@ -0,0 +1,25 @@
+#include "AMControlConnectedLEDView.h"
+
+AMControlConnectedLEDView::AMControlConnectedLEDView(AMControl *control, QWidget *parent) :
+	AMControlLEDView(control, parent)
+{
+
+}
+
+AMControlConnectedLEDView::~AMControlConnectedLEDView()
+{
+
+}
+
+void AMControlConnectedLEDView::update()
+{
+	if (control_) {
+		if (control_->isConnected())
+			setAsGreenOn();
+		else
+			setAsRedOn();
+
+	} else {
+		setAsGreenOff();
```

Comment:
I'd imagine if you were a red light the red off would be the one you wanted? If it looks better with the green off then by all means keep it as it is now.

Sent from my iPhone

On Dec 22, 2015, at 11:32 AM, helfrij <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/ui/beamline/AMControlConnectedLEDView.cpp<https://github.com/acquaman/acquaman/pull/1803#discussion_r48276920>:

> +
> +AMControlConnectedLEDView::~AMControlConnectedLEDView()
> +{
> +
> +}
> +
> +void AMControlConnectedLEDView::update()
> +{
> +     if (control_) {
> +             if (control_->isConnected())
> +                     setAsGreenOn();
> +             else
> +                     setAsRedOn();
> +
> +     } else {
> +             setAsGreenOff();


What do you suggest?

-
Reply to this email directly or view it on GitHub<https://github.com/acquaman/acquaman/pull/1803/files#r48276920>.